continuous-integration/readme.md

```diff
@@ -1,3 +1,236 @@
 # Continuous Integration
 
-Please see [CI/CD](CICD.md) while this page is awaiting contributions.
+![image](https://user-images.githubusercontent.com/7635865/76624154-c2c12800-6502-11ea-912d-a260c821ac41.png)
+
+We encourage engineering teams to make an upfront investment during Sprint 0 of a project to establish an automated and repeatable pipeline which continuously integrates code and releases system executable(s) to target cloud environments. Each integration should be verified by an automated build process that asserts a suite of validation tests pass and surface any errors across the developer team.
+
+We encourage teams to implement the CI/CD pipelines before any service code is written for customers, which usually happens in Sprint 0(N). This way, the engineering team can develop and test their work in isolation without impacting other developers and promote a consistent devops workflow throughout the engagement.
+
+These [principles](https://martinfowler.com/articles/continuousIntegration.html) map directly agile software development lifecycle [practices](https://en.wikipedia.org/wiki/Agile_software_development).
+
+<!-- markdownlint-disable MD036 -->
+**Table of contents**
+<!-- markdownlint-enable MD036 -->
+
+- [Continuous Integration](#continuous-integration)
```

Comment:
A few other sections have used TOC but only with the level 2 titles.  We should probably standardize this so all sections look the same, either if this also moves to only level 2 or if all sections display all.

We can open a discussion in the main teams about this unless you want to change this to level 2 only

Reply:
I'll switch to level 2 titles as the leaf node of the TOC